db/migrate/20140908072051_devise_create_users.rb

```diff
@@ -1,42 +0,0 @@
-class DeviseCreateUsers < ActiveRecord::Migration
```

Comment:
Basically the idea is the following:
1) delete all our old migrations
2) start our schema.rb file from their existing db schema
3) migrate forward until it matches what we had before